text/0000-read-by-prefix.md

```diff
@@ -0,0 +1,111 @@
+- Feature Name: read_by_prefix
+- Start Date: 2018-12-19
+- RFC PR:
+- Sawtooth Issue:
+
+# Summary
+[summary]: #summary
+
+This RFC proposes an additional API to transaction context,
+context.listAddresses(address_prefix), which will get address prefix as input
```

Comment:
This should probably return an iterator so not all addresses must be obtained at once.

Reply:
Agreeed.